mlflow/models/evaluation/base.py

```diff
@@ -111,6 +111,8 @@ def eval_fn(
         version: (Optional) The metric version. For example ``v1``.
         metric_details: (Optional) A description of the metric and how it is calculated.
         metric_metadata: (Optional) A dictionary containing metadata for the metric.
+        genai_metric_args: (Optional) A dictionary containing arguments required to call
+            make_genai_metric or make_genai_metric_from_prompt.
```

Comment:
```suggestion
        genai_metric_args: (Optional) A dictionary containing arguments specified by users
            when calling make_genai_metric or make_genai_metric_from_prompt. Those args
            are persisted so that we can deserialize the same metric object later.
```